### PR TITLE
PLAT-894: Server tracing and Logging context

### PR DESCRIPTION
**Pull request by khouari1**
Thank you for contributing to the documentation.

Your Jira ticket is:
https://genesisglobal.atlassian.net/browse/PLAT-894

Have you provided changes for all the relevant versions: next, 2022.4, 2023.1, etc?
Yes

Have you checked all new or changed links?
Yes

Is there anything else you would like us to know?
I have added tracing above release notes in the sidebar

For reference: 

- if you are an internal contributor:
  - We have an [internal contributions guide](https://www.notion.so/genesisglobal/Contributing-new-documentation-75953fb245f246ff872789035451a0c4)
  - We have a [style guide](https://www.notion.so/genesisglobal/Documentation-style-guide-5b04ec6fe12f4262b90d192effd8059b) 
- If you are an external contributor:
  - We have an [external contribution guide](../Type-of-contribution)

**This week's exciting excerpt from the style guide**
LISTS. Only use numbered lists when the sequence is important. Usually, that is a set of instructions.
For example:
1. Change the dictionary files.
2. Run genesisInstall.
3. Run remap.
In all other cases, use an **unnumbered** list. OK?

### CODE REVIEW

File: docs/05_operations/11_tracing/01_tracing.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+title: 'Tracing'
+sidebar_label: 'Tracing'
+id: tracing
+keywords: [operations, tracing]
+tags:
+    - operations
+    - tracing
+---
+
+The Genesis Tracing module enables you to capture tracing for your application through its integration with [OpenTelemetry](https://opentelemetry.io/). OpenTelemetry is a vendor-neutral open-source observability freamework and has become an industry-standard.
```
Comment (AliAlbaity) [inline docs/05_operations/11_tracing/01_tracing.md:11]: I'm guessing you meant framework not freamework

Comment (khouari1) [reply docs/05_operations/11_tracing/01_tracing.md:11]: Yep, thanks!

**Comment by GuiWar** [comment]
Hi @khouari1 . is this a work still in progress?

**Comment by khouari1** [comment]
> Hi @khouari1 . is this a work still in progress?

Yes. The server code hasn't been merged in yet.

**Comment by khouari1** [comment]
Opened a new PR to resolve conflicts easier

**Pull request closed without merging**